controller/src/main/java/io/pravega/controller/server/eventProcessor/requesthandlers/SealStreamTask.java

```diff
@@ -59,6 +68,17 @@ public SealStreamTask(final StreamMetadataTasks streamMetadataTasks,
                         throw new TaskExceptions.StartException("Seal stream task not started yet.");
                     }
                 })
+                .thenCompose(x -> cancelTransactions(context, scope, stream)
+                        .thenAccept(cancelled -> {
+                            if (!cancelled) {
```

Comment:
Ok, the logic might be fine, but the choice of words is confusing. The aborts for the remaining txns are triggered, but they are not necessary complete, so it is not that they are not cancelled, it is just that aborting is in progress. If my interpretation is right, then we should possibly use `aborting` such that if the result is `true`, then we execute the if block, and otherwise we skip it.